vagrant/provision.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+sudo apt-get update
+## INSTALL dependencies
+# Install MySQL Server in a Non-Interactive mode. Default NO PASSWORD for root
+sudo DEBIAN_FRONTEND=noninteractive apt-get install -y redis-server mysql-server libmysqlclient-dev libmysqld-dev
+
+sudo apt-get install -y python2.7 python-virtualenv python-dev build-essential libbz2-dev libfreetype6-dev libgdbm-dev libxft-dev curl
+sudo apt-get install -y libjpeg-dev libldap2-dev libltdl-dev libmysqlclient-dev libreadline-dev libsasl2-dev libsqlite3-dev
+sudo apt-get install -y libssl-dev libxslt1-dev ncurses-dev zlib1g-dev libmemcached-dev
+sudo apt-get install -y libcap2-bin
+sudo apt-get install -y vim-nox git
+
+
+# INSTALL Ralph
+virtualenv --clear /home/vagrant
+/home/vagrant/bin/pip install -U pip
+/home/vagrant/bin/pip install -U setuptools==3.6
+virtualenv ralph
+. bin/activate
+
+cd sources/
+make install-dev
+
+# CREATE db
+./vagrant/provisioning_scripts/init_mysql.sh
+
+# POST provisioning fixes
+# Add automatic activation to user .profile file
+echo 'source /home/vagrant/bin/activate' >> /home/vagrant/.profile
```

Comment:
probably it isn't the only one thing which we want to add to `.profile` so could you create additional text file which then will be appended to `.profile` (for example `cat vagrant/profile_extensions >> /home/vagrant/.profile`)?